addons/website/views/snippets/s_company_team_basic.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_company_team_basic" name="Team">
+    <section class="s_company_team o_colored_level o_cc o_cc1 pt64 pb64" data-snippet="s_company_team_basic" data-name="Team">
+        <div class="container">
+            <div class="row o_grid_mode" data-row-count="10" style="column-gap: 32px;">
+                <div class="o_grid_item g-col-lg-12 g-height-2 col-lg-12" style="grid-area: 1 / 1 / 3 / 13; --grid-item-padding-y: 32px; --grid-item-padding-x: 0px; z-index: 5;">
+                    <h3 class="o_default_snippet_text" style="text-align: center;">Meet our experts</h3>
+                </div>
+                <div class="o_grid_item g-col-md-6 g-col-lg-3 g-height-8 col-md-6 col-lg-3" data-name="Team Member" style="grid-area: 3 / 1 / 11 / 4; --grid-item-padding-y: 16px; --grid-item-padding-x: 16px; z-index: 1; text-align: center;">
+                    <div class="o_not_editable" contenteditable="false">
+                        <img alt="" class="o_editable_media img-fluid rounded-circle w-lg-100" src="/web/image/website.s_company_team_image_1"/>
+                    </div>
+                    <p>
+                        <br/>
+                    </p>
+                    <h4 class="h5-fs">Tony Fred, CEO</h4>
+                    <p class="text-800">Chief Executive Officer</p>
+                </div>
+                <div class="o_grid_item g-col-md-6 g-col-lg-3 g-height-8 col-md-6 col-lg-3" data-name="Team Member" style="grid-area: 3 / 4 / 11 / 7; --grid-item-padding-y: 16px; --grid-item-padding-x: 16px; z-index: 2; text-align: center;">
+                    <div class="o_not_editable" contenteditable="false">
+                        <img alt="" class="o_editable_media img-fluid rounded-circle w-lg-100" src="/web/image/website.s_company_team_image_2"/>
+                    </div>
+                    <p>
+                        <br/>
+                    </p>
+                    <h4 class="h5-fs">Mich Stak, COO</h4>
+                    <p class="text-800">Chief Operational Officer</p>
```

Comment:
Color prestes don't currently handle `text-[grayscale level]` classes.
Use `text-muted` instead 😉

| this pr: `text-800` | proposal: `text-muted` |
|--------|--------|
| <img width="275" alt="image" src="https://github.com/user-attachments/assets/253392da-4340-4c14-887b-50077f2d2a52"> | <img width="266" alt="image" src="https://github.com/user-attachments/assets/a64dea20-1096-41df-96c7-524bd7fb2c65"> |

Reply:
If i'm not mistaken, we can't set `text-muted` via the editor, so I think we can't do that

Reply:
> If i'm not mistaken, we can't set `text-muted` via the editor, so I think we can't do that either

Sorry I discussed this on discord with Stefano, we said that even if the user can't set it up it's fine to propose it as default since the user is able to change it if he doesn't want this result. (even tho he can't easily reproduce text-muted)

Reply:
That's the point, we are constrained to stick to what the editor can do

https://github.com/odoo/odoo/pull/177546#discussion_r1734362788 
and 
https://github.com/odoo/odoo/pull/177546#discussion_r1735736163